### PR TITLE
fastlane blogpost

### PR DESCRIPTION
**Pull request by birmacher**

### CODE REVIEW

File: source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+authors: Barnabas Birmacher|birmacher@bitrise.io
+---
+
+![fastlane](fastlane_logo.png)
+
+We are really happy to let you know that we officially support [fastlane](https://fastlane.tools)!
+
+## What is fastlane?
+
+`Fastlane` lets you define and run your deployment pipelines for different environments. It helps you unify and automate your app’s release process. `Fastlane` connects all `fastlane tools` and third party tools, like CocoaPods and xctool.
```
Comment (viktorbenei) [inline source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown]: `fastlane` - preferred with lowercase `f`

File: source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* **frameit** quickly puts your screenshots into the right device frames 
+* **PEM** automatically generates and renew your push notification profiles 
+* **sigh** automatically generates and manages your provisioning profiles
+* **produce** creates new iOS apps on iTunes Connect and Developer Portal
+* **cert** automatically creates and maintain iOS code signing certificates 
+* **codes** creates promo codes for iOS Apps using the command line
+
+## How to get started?
+
+We have created a tutorial to easily get you started using `fastlane`.
```
Comment (viktorbenei) [inline source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown]: ..to make it easy for you to get started (?)

File: source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown (hunk 1 above)
Comment (birmacher) [inline source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown]: igen de mondat eleje

Comment (viktorbenei) [inline source/2015-08-05-introducing-fastlane-on-bitrise.html.markdown]: here

**Pull request merged**